mobile/src/main/java/org/hawkular/client/android/fragment/AlertsFragment.java

```diff
@@ -397,6 +401,8 @@ private void tearDownState(Bundle state) {
     public void onDestroyView() {
         super.onDestroyView();
 
+        isAlertsFragmentAvailableForAlerts = false;
```

Comment:
@janakamarasena Even though this variable is used only in case of alerts but still it is not bound to that. Its generic work is just to tell if UI exists or not. Thats it